From b8e32ffaaba7e95637d6a25b5cfa5a21a3ceb89a Mon Sep 17 00:00:00 2001
From: Richard <hobbes1069@gmail.com>
Date: Mon, 27 Jan 2014 12:34:43 -0600
Subject: [PATCH 02/11] Add channel callsign to OSD

After adding a HDHR Prime to my system it became apparent that the OSD did not display the channel callsign which made browsing channels very difficult. This seemed to work well for me. The SD theme could probably use this too but the widget placement would need to be tweaked.
---
 mythtv/themes/MythCenter-wide/osd.xml |    5 +++++
 1 file changed, 5 insertions(+)

diff --git a/mythtv/themes/MythCenter-wide/osd.xml b/mythtv/themes/MythCenter-wide/osd.xml
index a230606..1d4761d 100644
--- a/mythtv/themes/MythCenter-wide/osd.xml
+++ b/mythtv/themes/MythCenter-wide/osd.xml
@@ -219,6 +219,11 @@
             <align>hcenter,vcenter</align>
             <value>%CHANNUM%</value>
         </textarea>
+        <textarea name="callsign">
+            <area>925,10,300,80</area>
+            <font>large</font>
+            <align>allcenter</align>
+        </textarea>
         <imagetype name="iconpath">
             <area>1015,10,125,80</area>
         </imagetype>
-- 
1.7.10.2

